### DISCUSSION TITLE
Idea: Help in Migration from one test library to another

### DISCUSSION THREAD

**Discussion by poudelprakash** [antiwork/shortest#73] [Ideas]
I have a old client project in rails which has minitest and rspec, they want to shift to using rspec only.

On different context, There might be many repositories out there who are using outdated version of test libraries and instead of upgrading might even planning to switch to different library altogether.

AI based migration would also be helpful for such scenarios.

Would that kind of scenario be good use case for shortest or that better be in some fork?

I am thinking these as some preset of workflows:
for eg:
Minitest to rspec migration would be 1 preset, 
I would select this workflow, provide source folder(ai auto suggesting the folder) and providing destination folder.
Then shortest would convert the tests and send PR

**Comment by slavingia** [comment]
Don't think it's a fit for now
